tests/phpt/expanders/nested_ast_expander.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Provides an AST to a custom expander
+--FILE--
+<?php
+
+$(macro) {
+    $(\Yay\tests\fixtures\parsers\my_nested_ast_parser())
```

Comment:
Can we have this parser inlined here and removed from `fixtures/parsers.php`? These fixtures are there to test fully qualified parser names, we don't need to test it again here, so no need for the indirection in the test.